data/transactions/logic/TEAL_opcodes.md

```diff
@@ -728,76 +766,81 @@ When A is a uint64, index 0 is the least significant bit. Setting bit 3 to 1 on
 - Stack: ..., A: []byte, B: uint64, C: uint64 &rarr; ..., []byte
 - Copy of A with the Bth byte set to small integer (between 0..255) C. If B is greater than or equal to the array length, the program fails
 - Availability: v3
 
-## extract s l
+## extract
 
-- Opcode: 0x57 {uint8 start position} {uint8 length}
+- Syntax: `extract S L` ∋ S: start position, L: length
+- Bytecode: 0x57 {uint8}, {uint8}
 - Stack: ..., A: []byte &rarr; ..., []byte
 - A range of bytes from A starting at S up to but not including S+L. If L is 0, then extract to the end of the string. If S or S+L is larger than the array length, the program fails
 - Availability: v5
 
 ## extract3
 
-- Opcode: 0x58
+- Bytecode: 0x58
 - Stack: ..., A: []byte, B: uint64, C: uint64 &rarr; ..., []byte
 - A range of bytes from A starting at B up to but not including B+C. If B+C is larger than the array length, the program fails<br />`extract3` can be called using `extract` with no immediates.
 - Availability: v5
 
 ## extract_uint16
 
-- Opcode: 0x59
+- Bytecode: 0x59
 - Stack: ..., A: []byte, B: uint64 &rarr; ..., uint64
 - A uint16 formed from a range of big-endian bytes from A starting at B up to but not including B+2. If B+2 is larger than the array length, the program fails
 - Availability: v5
 
 ## extract_uint32
 
-- Opcode: 0x5a
+- Bytecode: 0x5a
 - Stack: ..., A: []byte, B: uint64 &rarr; ..., uint64
 - A uint32 formed from a range of big-endian bytes from A starting at B up to but not including B+4. If B+4 is larger than the array length, the program fails
 - Availability: v5
 
 ## extract_uint64
 
-- Opcode: 0x5b
+- Bytecode: 0x5b
 - Stack: ..., A: []byte, B: uint64 &rarr; ..., uint64
 - A uint64 formed from a range of big-endian bytes from A starting at B up to but not including B+8. If B+8 is larger than the array length, the program fails
 - Availability: v5
 
-## replace2 s
+## replace2
 
-- Opcode: 0x5c {uint8 start position}
+- Syntax: `replace2 S` ∋ S: start position
+- Bytecode: 0x5c {uint8}
 - Stack: ..., A: []byte, B: []byte &rarr; ..., []byte
 - Copy of A with the bytes starting at S replaced by the bytes of B. Fails if S+len(B) exceeds len(A)<br />`replace2` can be called using `replace` with 1 immediate.
 - Availability: v7
 
 ## replace3
 
-- Opcode: 0x5d
+- Bytecode: 0x5d
 - Stack: ..., A: []byte, B: uint64, C: []byte &rarr; ..., []byte
 - Copy of A with the bytes starting at B replaced by the bytes of C. Fails if B+len(C) exceeds len(A)<br />`replace3` can be called using `replace` with no immediates.
 - Availability: v7
 
-## base64_decode e
+## base64_decode
 
-- Opcode: 0x5e {uint8 encoding index}
+- Syntax: `base64_decode E` ∋ E: [base64](#field-group-base64)
+- Bytecode: 0x5e {uint8}
 - Stack: ..., A: []byte &rarr; ..., []byte
 - decode A which was base64-encoded using _encoding_ E. Fail if A is not base64 encoded with encoding E
 - **Cost**: 1 + 1 per 16 bytes of A
 - Availability: v7
 
-`base64` Encodings:
+### Field Group base64
+
+Encodings
```

Comment:
This is an example where I think "Field Group" is confusing, and the old
`base64 Encodings`
is nicer.
Maybe that means the anchor names, like on line 834, should also be customized by the FieldGroup name, so this one would be `#base64-encodings`.